admin/settings/advanced.mdx

```diff
@@ -1,17 +1,17 @@
 ---
-title: "Advanced"
+title: "Ingestion Controls"
 ---
 
-The Advanced section is where users can tailor their log tracking infrastructure. The Middleware Host Agent will search for changes in your account configuration and automatically update the configuration YAML file and reload itself.
+The Ingestion Controls section is where users can tailor their log tracking infrastructure. The Middleware Host Agent will search for changes in your account configuration and automatically update the configuration YAML file and reload itself.
 
 Fine-tune your log collection process by selecting your preferred hosts for tracking infrastructure file-based logs, Middleware Agent Logs, and specify custom directory paths for monitoring your log collection.
 
-<Warning> This feature is not available in the Docker Agent since Docker does not allow dynamic volume bindings. </Warning>
+<Note> This feature is not available in the Docker Agent since Docker does not allow dynamic volume bindings. </Note>
```

Comment:
I believe this note is just about changing the directory path for logs. We should move this so that it's clear as such